scripts/datasets/question_answering/README.md

```diff
@@ -15,3 +16,62 @@ squad
 ├── train-v{version}.json
 ├── dev-v{version}.json
 ```
+## SearchQA
+Following BSD-3-Clause License, we uploaded the SearchQA to our S3 bucket and provide the link to download the processed txt files. Please check out the [Google drive link](https://drive.google.com/drive/u/0/folders/1kBkQGooNyG0h8waaOJpgdGtOnlb1S649) to download to raw and split files collected through web search using the scraper from [GitHub repository](https://github.com/nyu-dl/dl4ir-searchQA).
+
+Directory structure of the searchqa dataset will be as follows
+```
+searchqa
+├── train.txt
+├── val.txt
+├── test.txt
+```
+
+## TriviaQA
+[TriviaQA](https://nlp.cs.washington.edu/triviaqa/) is an open domain QA dataset. See more useful scripts in [Offical Github](https://github.com/mandarjoshi90/triviaqa)
+
+Run the following command to download triviaqa
+
+```bash
+python prepare_triviaqa.py --version rc         # Download TriviaQA version 1.0 for RC (2.5G)
+python prepare_triviaqa.py --version unfiltered # Download unfiltered TriviaQA version 1.0 (604M)
+```
```

Comment:
Besides mentioning that we are able to download via
```bash
python prepare_triviaqa.py --version rc         # Download TriviaQA version 1.0 for RC (2.5G)
python prepare_triviaqa.py --version unfiltered # Download unfiltered TriviaQA version 1.0 (604M)
```

We can also mention that we can download with
```bash
nlp_data prepare_triviaqa --version rc
nlp_data prepare_triviaqa --version unfiltered
```